x/storage/abci.go

```diff
@@ -13,6 +13,7 @@ func BeginBlocker(ctx sdk.Context, keeper k.Keeper) {
 		keeper.ClearDiscontinueObjectCount(ctx)
 		keeper.ClearDiscontinueBucketCount(ctx)
 	}
+	keeper.InitDeleteInfo(ctx)
```

Comment:
Why needs store this empty key if there is no deleted resource in this block.

Reply:
yeah actually this is redundant, because when appending resource ids to deleteInfo, we are still checking the key existence.